### PR TITLE
Toggle answer

### PR DESCRIPTION
**Pull request by Ju-Schn**
Add button to toggle answer, toggle logic (custom hook), update styling and card.spec

### CODE REVIEW

File: src/components/Card.js [hunk 1]
```diff
@@ -1,14 +1,55 @@
...
+  const [solution, toggleSolution] = useToggle(false);
   return (
     <FileCard>
-      <h2>Frage: </h2>
-      <span>{question}</span>
-      <h2>Antwort: </h2>
-      <span>{answer}</span>
+      <ScreenReaderOnly>
+        <h2>Frage: </h2>
+      </ScreenReaderOnly>
```
Comment (MerleMak) [inline src/components/Card.js:12]: do you want to use the h2 as label for the styled question? What is the utility of it? Maybe you could connect it to the question with an "ariaLabelledBy"

**Review by MerleMak — APPROVED** [review]
Amazing work! You are really fast and should be proud of yourself :)

**Comment by Ju-Schn** [comment]
> Amazing work! You are really fast and should be proud of yourself :)

Thank you, your code is reeeeaaally great, too!

File: src/components/Card.js (hunk 1 above)
Comment (Ju-Schn) [reply src/components/Card.js:12]: > do you want to use the h2 as label for the styled question? What is the utility of it? Maybe you could connect it to the question with an "ariaLabelledBy"

I wasn't sure if it's common to label a span, I think like this it's read in normal textflow as if it was displayed. I want to test it in the evening :)

**Pull request merged**